src/Nethermind/Nethermind.BeaconNode.Host/Development9001/appsettings.json

```diff
@@ -1,13 +1,36 @@
 {
+  "Environment": "Development",
+  "Kestrel": {
+    "Endpoints": {
+      "Http": {
+        "Url": "http://localhost:5001"
+      },
+      "HttpsDefaultCert": {
+        "Url": "https://localhost:8231"
+      }
+    }
+  },
+  "Peering": {
+    "Port": 9001
+  },
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug",
+      "System": "Information",
+      "Microsoft": "Information",
+      "Microsoft.AspNetCore": "Warning",
+      "Nethermind.BeaconNode.BeaconStateTransition": "Information"
+    }
+  },
   "BeaconChain": {
```

Comment:
we will need some way to quickly load various configs here like in Nethermind-Eth1, definitely not a priority now
and I see that it may be supported out of the box by the ASP settings files here